writings/2022-06-20-mev-boost-call-for-testing.mdx

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
post needs an OG image, like https://github.com/flashbots/flashbots-writings-website/blob/main/writings/2022-06-07-why-run-mevboost.mdx

Reply:
What's OG?

Reply:
opengraph, it's the image that's shown when sharing a post. https://ogp.me/ and see also https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started#opengraph

Reply:
I'm very bad at making things look nice. Is there a place to get good opengraph images?
Until we have a designer assigned to our communications, I would prefer to just go with whatever is the default on the writings website.

Reply:
You can just use any free image, but without it the preview will not look good, there's no default

Reply:
- [https://pixabay.com/](https://pixabay.com)
- [https://unsplash.com/](https://unsplash.com/)